src/System.Reflection.Primitives/src/System/Reflection/CallingConventions.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+////////////////////////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////////////////////////
+//
+// CallingConventions is a set of Bits representing the calling conventions
+// 
+
+//    in the system.
+//
+// <EMAIL>Author: meichint</EMAIL>
```

Comment:
It  looks like you need to run the scrubing tool on this as well as that will usually take emails out.